### ISSUE TITLE
Warning for outdated/insecure PrivateBin versions

### ISSUE THREAD

**Issue by rugk** [PrivateBin/WebExtension#10]
We should also add a field to the JSON code to list depreciated/insecure PrivateBin versions, so that the user can also be warned if such a version is used. This may of course be an enhancement for later times, but we can already keep it in mind when implementing stuff here.

Proposed way to work:
1. We update the JSON with new hashes and add the current version as a depreciated version.
2. We publish the new PrivateBin server version and urge web admins to update.
3. The add-on gets the new JSON and displays warnings if a PrivateBin version is outdated
4. We may later change this depreciated to a "insecure" PrivateBin version when we think server admins had enough time to update, so now an even scarier warning is shown

**Comment by gjedeer** [comment]
5. Keep "insecure" label only for releases which actually affect security - I don't follow the development of the rest of the code but I guess not every release fixes security issues

**Comment by rugk** [comment]
Yeah, only when a release is security-critical the process should be triggered at all (starting with 1.).
